runtime/runtime_test.go

```diff
@@ -0,0 +1,58 @@
+package runtime_test
+
+import (
+	"testing"
+	"time"
+
+	"github.com/hamba/statter"
+	"github.com/hamba/statter/runtime"
+	"github.com/stretchr/testify/mock"
+)
+
+func TestRuntime(t *testing.T) {
+	m := &mockComplexReporter{}
+	m.On("Gauge", mock.Anything, mock.Anything, mock.Anything)
+	m.On("Timing", mock.Anything, mock.Anything).Return(func(_ time.Duration) {})
```

Comment:
I don't like seeing `mock.Anything` used except in very specific cases :p What do you have to say for your defense! If the issue is that many different values are given and it would be impossible to list all of them, you should at least use [`mock.AnythingOfType(XXX)`](https://github.com/stretchr/testify/blob/master/mock/mock.go#L671-L678).

Reply:
In this case I generally dont care what what they are called with (This is extensively tested else where), cause I later look for a specific call, but you are correct, AnythingOfType would be better.